sf-docker/sf-observability/docker-compose.yml

```diff
@@ -0,0 +1,214 @@
+version: "3.6"
+
+services:
+  obsminio:
+    extends:
+      service: common
+      file: docker-compose-common.yml
+    image: "minio/minio:RELEASE.2022-06-02T02-11-04Z"
+    profiles: [singlenode, multinode-aggregator]
+    command: server /data --console-address ":9001"
+    expose:
+      - 9000
+    ports:
+      - "9901:9001"
+    volumes:
+      - minio_data:/data
+    networks:
+      observability:
+    restart: unless-stopped
+
+  obsminio_createbuckets:
+    extends:
+      service: common
+      file: docker-compose-common.yml
+    image: minio/mc
+    profiles: [singlenode, multinode-aggregator]
+    depends_on:
+      obsminio:
+        condition: service_started
+    entrypoint: >
+      /bin/sh -c "
+      sleep 10;
+      /usr/bin/mc config host add obsminio http://obsminio:9000 minioadmin minioadmin;
+      /usr/bin/mc mb obsminio/loki;
+      /usr/bin/mc policy set public obsminio/loki;
+      exit 0;
+      "
+    networks:
+      observability:
+  
+  grafana:
+    extends:
+      service: common
+      file: docker-compose-common.yml
+    profiles: [singlenode, multinode-aggregator]
+    image: grafana/grafana:10.1.4
+    ports:
+      - 9900:3000
+    volumes:
+      - ./configuration/grafana/datasources:/etc/grafana/provisioning/datasources
+      - ./configuration/grafana/dashboards:/etc/grafana/provisioning/dashboards
+      - grafana_data:/var/lib/grafana
+    environment:
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
+      - GF_AUTH_DISABLE_LOGIN_FORM=true
+    networks:
+      observability:
+    restart: unless-stopped
+  
+  loki:
+    extends:
+      service: common
+      file: docker-compose-common.yml
+    image: grafana/loki:2.9.1
+    profiles: [singlenode, multinode-aggregator]
+    depends_on:
+      obsminio:
+        condition: service_started
+      obsminio_createbuckets:
+        condition: service_completed_successfully
+    volumes:
+      - ./configuration/loki:/etc/loki
+      - loki_data:/loki
+    command: -config.file=/etc/loki/loki-config.yaml
+    expose:
+      - 3100
+    networks:
+      observability:
+    restart: unless-stopped
+  
+  alloy:
+    extends:
+      service: common
+      file: docker-compose-common.yml
+    image: grafana/alloy:v1.5.0
+    profiles: [singlenode, multinode-scraper]
+    entrypoint: [ "/bin/sh","-c" ]
+    command:
+      - |
+        export HOSTNAME=$$(cat /etc/hostname)
+        /bin/alloy run --server.http.listen-addr=0.0.0.0:12345 --storage.path=/var/lib/alloy/data /etc/alloy/config.alloy
+    environment:
+      - LOKI_URL=http://loki:3100/loki/api/v1/push
+      - PROMETHEUS_URL=http://prometheus:9090/api/v1/write
+    volumes:
+      - ./configuration/alloy/config.alloy:/etc/alloy/config.alloy
```

Comment:
Please include the config file using docker compose `configs`, also make sure to specify file attributes.

Reply:
done

Reply:
done